README.md

```diff
@@ -169,14 +170,41 @@ If you're using Maven, this is how your `pom.xml` should look like:
 </project>
 ```
 
-With this configutation you can run it from command line:
+With this configuration you can run it from command line:
 
 ```
 $ mvn clean integration-test -Phit-refresh -Dport=8080
 ```
 
 Maven will start the server and you can see it at `http://localhost:8080`.
 
+## Build and Run With Gradle
+
+If you're using Gradle, this is how your `build.gradle` should look like:
+
+```
```

Comment:
@paulodamaso you can specify language for code blocks in Github markdown to enable keyword highlighting: ` ```gradle` (see https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting)